modules/ROOT/pages/apply-configure-xml-threat-task.adoc

```diff
@@ -9,31 +9,34 @@ endif::[]
 >s|Policy Name|XML Threat Protection
 >s|Summary|Protects against malicious XML in API requests
 >s|Category|Security
->s|Since Mule Version|3.8.0
+>s|Available from Mule version|3.8.0
 >s|Returned Status Codes|400 - Bad Request
 |===
 
-== Purpose
 
-XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.
+Applications processing XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes, and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.
```

Comment:
```suggestion
Applications processing XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes, and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic increases in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.
```
Is "the XML threat protection policy" the same as "the JSON Threat Protection policy"?
If so, try to stick with just one name.
If not...be sure that the first is a common noun ("a XML threat protection policy") or a proper noun, like the JSON version.

Reply:
Done! Thanks!